user-guide/Advanced_Modules/DOM/DOM_best_practices.md

```diff
@@ -0,0 +1,200 @@
+---
+uid: DOM_best_practices
+---
+
+# DOM best practices
+
+This document outlines best practices for both DOM model design and the performance implications of these design decisions. It is strongly recommended to consider these guidelines when designing a DOM model.
+
+In short, the following is important:
+
+- **Group DomDefinitions:** Consolidate related `DomDefinitions` within the same module to optimize database performance and minimize resource usage.
+
+- **Disable Instance History:** Turn off unnecessary instance history tracking to conserve resources, especially beneficial for DOM managers with frequent create or update actions.
+
+- **Compact DomInstances:** Prioritize essential data, avoid duplication, and limit unnecessary fields to keep `DomInstances` small and efficient.
+
+- **Optimize CRUD Calls:** Use bulk operations for create, update, or delete actions, and avoid multiple updates to the same `DomInstance` in the same script to reduce overhead.
+
+- **Efficient Scripting:** Design scripts for fast and efficient execution, particularly focusing on CRUD scripts, to minimize execution time and potential script queues.
+
+## Module and settings
+
+### Group DomDefinitions that belong to the same solution within the same DOM module
+
+It is recommended to limit the number of DOM modules on a DataMiner system. Each DOM module has its separate indexes in the Elasticsearch or OpenSearch database. These require a certain amount of heap memory and add to the number of active 'shards' in the database. It is thus recommended to define all `DomDefinitions` and store all `DomInstances` within the same DOM module instead of creating a DOM module for each separate DOM model. However, if a certain model is completely stand-alone and is reused by other solutions, it can still be advisable to keep them separate. It is recommended to keep the number of DOM modules on one system well below 50. This number is, however, dependent on the number of DB nodes and their resource allocation. If STaaS is used, the performance aspect is less relevant due to the different way the DOM data is stored.
```

Comment:
S: drop the quotes on shards  but maybe reference some basic documentation (e.g. <https://en.wikipedia.org/wiki/Shard_(database_architecture)>)

Reply:
Rephrased it a bit so it is less technical.